### ISSUE TITLE
Inquiry about comparison experiments and pretrained weights

### ISSUE THREAD

**Issue by Masaaki-75** [isbrycee/T-Mamba#1]
Hi. Thanks for open-sourcing the code!  

The superior performance reported in the paper makes T-Mamba fairly promising, useful and interesting. I am curious about the following aspects and greatly appreciate your insights on them:
- Do you have a plan to make the pretrained weights public? Can't wait to try 🤩! 
- How much of a performance advantage will T-Mamba have over nnUNet? I used to try nnUNet on medical image segmentation and found it performed surprisingly excellent over many models (including some SOTA ones).

**Comment by isjinghao** [comment]
Hi, thanks for your interest.

The pre-trained weights on 3D CBCT will be public in nearly three weeks. We are conducting more experiments and analysis on 3D CBCT and 2D X-ray images, and will update the whole manuscript.

The code, pre-trained weights, and datasets will be fully available. 

Regarding the performance on nnUNet, we will implement this experiment and declare the result. 

If u have any other problems, feel free open an issue.

**Comment by Masaaki-75** [comment]
Got it. Thank you for your reply!
